integrated_channels/integrated_channel/tasks.py

```diff
@@ -89,6 +90,51 @@ def _log_batch_task_finish(task_name, channel_code, job_user_id,
         ))
 
 
+@shared_task
+@set_code_owner_attribute
+def mark_orphaned_content_metadata_audit():
+    """
+    Placeholder
+    """
+    start = time.time()
+
+    linked_metadata_audits = ContentMetadataItemTransmission.objects.none()
+    # Go over each integrated channel
+    for individual_channel in INTEGRATED_CHANNEL_CHOICES.values():
+        try:
+            # Iterate through each configuration for the channel
+            for config in individual_channel.objects.all():
```

Comment:
I think we want to keep this on all content and not just active? We might need to make sure that orphaned records fetched later on in this task doesn't try and grab content for deactivated content.

Reply:
right, it would just skip over the entire channel/metadata set if the config is inactive